pkg/bindings/manifests/manifests.go

```diff
@@ -0,0 +1,127 @@
+package manifests
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/containers/image/v5/manifest"
+	"github.com/containers/libpod/libpod/image"
+	"github.com/containers/libpod/pkg/api/handlers"
+	"github.com/containers/libpod/pkg/bindings"
+	jsoniter "github.com/json-iterator/go"
+)
+
+// Create creates a manifest for the given name.  Optional images to be associated with
+// the new manifest can also be specified.  The all boolean specifies to add all entries
+// of a list if the name provided is a manifest list.  The ID of the new manifest list
+// is returned as a string.
+func Create(ctx context.Context, names, images []string, all *bool) (string, error) {
+	var idr handlers.IDResponse
+	conn, err := bindings.GetClient(ctx)
+	if err != nil {
+		return "", err
+	}
+	if len(names) < 1 {
```

Comment:
Is an empty name legal?  Do you need to test for not being ""?

Reply:
in cli , not here